docs/NSI_MATCHING.md

```diff
@@ -0,0 +1,32 @@
+## NSI Matching
+
+We attempt to match Items against [NSI](https://nsi.guide/).
+This is done automatically in `ApplyNSICategoriesPipeline`, please don't manually set `nsi_id` as they aren't stable.
+
+When we successfully match, we apply `nsi_id` and the NSI attributes to the Item, not overriding the Spider given attributes.
```

Comment:
is Item and Spider capitalisation intentional?

Reply:
Kind of, because I'm specifically talking about the Python objects.